optimuspy.py

```diff
@@ -164,49 +164,46 @@ def main(instance_name: str, cube_name: str, view_name: str, process_name: str,
             write_vmm_vmt(tm1, cube_name, "1000000", "1000000")
 
             logging.info(f"Starting analysis for cube '{cube_name}'")
-            original_dimension_order = tm1.cubes.get_storage_dimension_order(cube_name=cube_name)
-            logging.info(f"Original dimension order for cube '{cube_name}' is: '{original_dimension_order}'")
+            storage_dimension_order = tm1.cubes.get_storage_dimension_order(cube_name=cube_name)
```

Comment:
original to me means the "display" order, while this is the storage order

Reply:
We use the term `OriginalOrder` also when presenting the results.
https://github.com/cubewise-code/optimus-py/issues/49

I think `original order` is a better term than `storage dimension order` especially when presenting the results.
Original would refer to the point in time before the optimization. I don't like the term storage dimension order for the initial order as the storage dimension order mutates over time.

I wonder if we can agree on an alternative term to use for the variable and in the output. 
How about `InitialDimensionOrder`?

Reply:
I think we should go with `InitialDimensionOrder`